test/e2e/components/vl-accordion.js

```diff
@@ -9,10 +9,6 @@ class VlAccordion extends VlElement {
         return this.shadowRoot.findElement(By.css('#accordion-toggle'));
     }
 
-    async _getContent() {
```

Comment:
Het opvragen van de content van een accordion kan dan nu helemaal niet meer, precies? Is het misschien niet de bedoeling dat er een publieke getContent is zodat afnemers kunnen testen op de content?

Reply:
Werd niet gebruikt, maar inderdaad misschien wel de bedoeling, anders moeten afnemers in de shadow dom gaan beginnen zoeken naar de content als ze die willen en dat is misschien niet echt de bedoeling?

Reply:
Het is inderdaad de bedoeling dat de de e2e test components enkele basiszaken aanbieden aan afnemers. Meestal wordt er dan ook een e2e test geschreven die er gebruik van maakt in de webcomponent zelf.